packages/docs/getting-started/running-a-validator.md

```diff
@@ -210,7 +212,7 @@ The `mine` flag does not mean the node starts mining blocks, but rather starts t
 
 The `networkid` parameter value of `1101` indicates we are connecting the Baklava Beta network.
 
-### Running the Attestation Service
+### Running the Attestation signer
```

Comment:
Service?

Reply:
Will do with the improved validator setup docs PR